Release `logzio logs collector` v2.2.1

ralongit opened the pull request:
## Description 
- Add SignalFx logs support

<!-- 
    "This PR [adds/removes/fixes/replaces] the [feature/bug/etc].."
    Do not leave this blank.
    If relevant, describe the previous behaviour compared to the behaviour post your code change.
-->

## What type of PR is this?
#### (check all applicable)
- [x] 🍕 Feature 
- [ ] 🐛 Bug Fix
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [x] ✅ Test
- [x] 🤖 Build / CI
- [ ] ⏩ Revert

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help from somebody

ralongit committed:
Add SignalFx logs

ralongit committed:
Update CHANGELOG.md

ralongit committed:
fix deployment and daemonset names

ralongit committed:
fail-fast: false & revert cm filter tests

ralongit committed:
update tests query

ralongit committed:
fix signalfx test

ralongit committed:
update dimension

ralongit committed:
simplify query

ralongit committed:
export env_id and update log type

ralongit committed:
update env

ralongit committed:
Update chart version & fix env id

ralongit committed:
remove extra spaces

ralongit committed:
Update tests env id

ralongit committed:
debug signalfx tests

ralongit committed:
Update signalfx_logs_e2e_test.go

ralongit committed:
Fix missing component label in deployment

yotamloe commented on charts/logzio-logs-collector/values.yaml:
```diff
@@ -225,7 +228,33 @@ config:
+      # SignalFx receiver to accept logs from SignalFx client libraries # https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/signalfxreceiver/README.md
+      signalfx:
+        endpoint: "0.0.0.0:9943"
+        access_token_passthrough: true
```
this param seems to be irelevant for logz

ralongit committed:
Remove irrelevant signalfx param

ralongit committed:
updates to config & tests

- Remove global.signalFx
- Simplify tests
- remove batch signalfx config, use base config
- add k8sattributes
- add log level
- add log type
